iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -6105,12 +6103,17 @@
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				BUNDLE_IDENTIFIER = app.omaps.debug;
 				CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES;
+				CLANG_UNDEFINED_BEHAVIOR_SANITIZER_NULLABILITY = NO;
+				CLANG_WARN_COMMA = NO;
```

Comment:
Please also revert this line. Zero-warnings policy is done by fixing the source of warnings, not by silencing them.